internal/worker/sshserver/server.go

```diff
@@ -149,6 +143,28 @@ func NewServerWorker(config ServerWorkerConfig) (worker.Worker, error) {
 	return s, nil
 }
 
+func (s *ServerWorker) NewJumpServer() *ssh.Server {
+	server := ssh.Server{
+		ConnCallback: s.connCallback(),
+		PublicKeyHandler: func(ctx ssh.Context, key ssh.PublicKey) bool {
+			return false
+		},
+		PasswordHandler: func(ctx ssh.Context, password string) bool {
+			return false
+		},
+		ChannelHandlers: map[string]ssh.ChannelHandler{
+			"direct-tcpip": s.directTCPIPHandler,
+		},
+	}
+
+	if flag.Lookup("test.v") != nil && s.config.disableAuth {
```

Comment:
Want to check that this is okay. We considered another option of using the options pattern, and allowing options to be passed in, with an option declared inside a _test.go file called DisableAuthOption(), but that get's overcomplicated quickly.

Reply:
Not a chance, not in production code.

Pass in a DisableAuthOption.

Reply:
Removed checking the flag.